.github/workflows/main_housingmanagementsystemapi.yml

```
@@ -23,6 +23,11 @@ jobs:
         run: dotnet tool update --global dotnet-format
       - name: Run dotnet format
         run: dotnet format --check --verbosity diagnostic
+      - uses: hashicorp/setup-terraform@v2
+      - name: Terraform fmt

```

The `name` specification is display text and it would be better to avoid abbreviations, i.e. 
```suggestion
      - name: Terraform Format
```

---

Done in [8ba50d0](https://github.com/Newarkandsherwood/HousingManagementSystemApi/pull/9/commits/8ba50d0b85e67675434435ea1a1df0b802236544)